spec/javascripts/live_search_spec.js

```diff
@@ -1,5 +1,8 @@
 describe('liveSearch', function () {
   var $form, $results, _supportHistory, liveSearch, $atomAutodiscoveryLink, $count
+  var GOVUK = window.GOVUK || {}
+  GOVUK.modules = GOVUK.modules || {}
+  GOVUK.modules.start = GOVUK.modules.start || function () {}
```

Comment:
I expect you can remove the need for this by adding test-dependencies.js to https://github.com/alphagov/finder-frontend/blob/d3b4771780258ea9250c44439f93428d64411ce1/spec/javascripts/support/jasmine.yml#L16

Reply:
@kevindew thanks for the suggestion, works perfectly, all updated.